mongodbatlas/resource_mongodbatlas_cluster.go

```diff
@@ -634,8 +642,8 @@ func resourceMongoDBAtlasClusterRead(ctx context.Context, d *schema.ResourceData
 		return diag.FromErr(fmt.Errorf(errorClusterSetting, "backup_enabled", clusterName, err))
 	}
 
-	if err := d.Set("provider_backup_enabled", cluster.ProviderBackupEnabled); err != nil {
```

Comment:
Since this is only deprecated should we leave these till removal?

Reply:
Yes ,I will roll back to keep it and validate if it's using deprecated or the new parameter